test/templates/table/component.njk

```diff
@@ -41,29 +41,29 @@ table:
 <div class="fd-table--fixed">
```

Comment:
I know you didn't write this, but can we do this cleaner? I guess it could be a followup but needing 2 extra divs when we have a fixed table seems weird to me.